RabbitMQ.pm

```diff
@@ -192,16 +216,10 @@ C<$arguments> is an optional hash which will be passed to the server
 when the queue is created.  This can be used for creating mirrored
 queues by using the x-ha-policy header.
 
-In scalar context, this method returns the queuename declared
-(important for retrieving the autogenerated queuename in the
-event that one was requested).
-
-In array context, this method returns three items: queuename,
-the number of message waiting on the queue, and the number
-of consumers bound to the queue.
-
 =head2 queue_bind($channel, $queuename, $exchange, $routing_key, $arguments)
 
+Bind the specified queue to the specified exchange with a routing key.
```

Comment:
More than just a routing key is supported. Maybe "given routing key and/or parameters"?